src/Raven.Server/Documents/PeriodicBackup/Retention/GlacierRetentionPolicyRunner.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Server.Documents.PeriodicBackup.Aws;
+
+namespace Raven.Server.Documents.PeriodicBackup.Retention
+{
+    public class GlacierRetentionPolicyRunner : RetentionPolicyRunnerBase
+    {
+        private readonly RavenAwsGlacierClient _client;
+
+        protected override string Name => "Glacier";
+
+        public GlacierRetentionPolicyRunner(RetentionPolicy retentionPolicy, string databaseName, Action<string> onProgress, RavenAwsGlacierClient client)
+            : base(retentionPolicy, databaseName, onProgress)
+        {
+            _client = client;
+        }
+
+        protected override Task<List<string>> GetFolders()
+        {
+            throw new NotSupportedException();
```

Comment:
Create a separate issue to support that.

Reply:
Done